src/resources/postcss/components/_loader.pcss

```diff
@@ -0,0 +1,93 @@
+.tribe-common {
+	/* -----------------------------------------------------------------------------
+	 *
+	 * Component: Loader
+	 *
+	 *
+	 * Example:
+	 * <div class="tribe-events-view-loader__dots tribe-common-c-loader">
+	 *    <div class="tribe-common-c-loader__dot tribe-common-c-loader__dot--first"></div>
+	 *    <div class="tribe-common-c-loader__dot tribe-common-c-loader__dot--second"></div>
+	 *    <div class="tribe-common-c-loader__dot tribe-common-c-loader__dot--third"></div>
+	 * </div>
+	 *
+	 * ----------------------------------------------------------------------------- */
+
+	/*.tribe-common-c-loader {}*/
+
+	.tribe-common-c-loader {
+		padding-top: calc(var(--spacer-11) * 3);
+
+		@media (--viewport-medium) {
+			padding-top: calc(var(--spacer-13) * 3 );
+		}
+	}
+
+	.tribe-common-c-loader__dot {
+		background-color: var(--color-accent-primary-background);
+		float: left;
```

Comment:
I'd suggest moving away from floats as much as possible for positioning. There are specific use cases for floats, such as floating an image with text wrapping it for example, but positioning is not it's best use case. It's a very old use case for floats and given browser support for things such as flex, we should probably be using other options instead.